### PR TITLE
Minor edits for the UG:Experiment-Management:Checkpoints

### PR DESCRIPTION
**Pull request by iesahin**
Moved project installation to `<details>`

Some typos fixed

### CODE REVIEW

File: content/docs/user-guide/experiment-management/checkpoints.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ With DVC experiments and checkpoints, you can:
 
 - Implement the best practice in deep learning to save your model weights as
   checkpoints.
-- Track all code and data changes corresponded to the checkpoints.
+- Track all code and data changes corresponded to a checkpoint.
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/checkpoints.md:10]: "The" was OK (refers to "as checkpoints" in the bullet above. But I think it should be "corresponding"

File: content/docs/user-guide/experiment-management/checkpoints.md [hunk 2]
```diff
@@ -19,12 +19,14 @@ custom Git reference.
...
-## Setting up the project
-
-This tutorial is going to cover how to implement checkpoints in a ML project
+This tutorial is going to cover how to implement checkpoints in an ML project
 using DVC. We're going to train a model to identify handwritten digits based on
 the MNIST dataset.
 
+<details>
+
+## 💡 Setting up the project
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/checkpoints.md:28]:
```suggestion
## ⚙️ Setting up the project
```

**Review by jorgeorpinel — APPROVED** [review]

File: content/docs/user-guide/experiment-management/checkpoints.md (hunk 1 above)
Comment (iesahin) [reply content/docs/user-guide/experiment-management/checkpoints.md:10]: I think we don't refer to a specific checkpoint, _the_ makes me to think _there is a single checkpoint for which we can associate all the code and data._ You're right it should be _corresponding_.

File: content/docs/user-guide/experiment-management/checkpoints.md (hunk 2 above)
Comment (iesahin) [reply content/docs/user-guide/experiment-management/checkpoints.md]: this didn't commit, I commit separately

**Pull request merged**